Leaderboards

wnashif opened the pull request:
Created a leaderboard view with various tabs for different leaderboards.
Added icons for the sidebar.
This was branched off of pr/19

derppancakes committed:
Merge branch 'main' of https://github.com/derppancakes/CS-471

derppancakes committed:
Merge branch 'main' of https://github.com/xPartyBear/CS-471

derppancakes committed:
Merge branch 'main' of https://github.com/xPartyBear/CS-471

derppancakes committed:
Merge branch 'main' of https://github.com/derppancakes/CS-471

derppancakes committed:
Merge branch 'main' of https://github.com/xPartyBear/CS-471

derppancakes committed:
Merge branch 'main' of https://github.com/derppancakes/CS-471

derppancakes committed:
Merge branch 'main' of https://github.com/xPartyBear/CS-471

derppancakes committed:
Merge branch 'main' of https://github.com/xPartyBear/CS-471

derppancakes committed:
further search functionality stuff

derppancakes committed:
Merge branch 'main' into SearchFunctionalityContinued

derppancakes committed:
ch ch ch changes!

derppancakes committed:
updated searchbar

wnashif committed:
Created leaderboards

Created the leaderboard page to be filled with data later.
Added icons for the sidebar.
Branched off of pr/19

derppancakes commented on frontend/src/views/LeaderboardsView.vue:
If you could, tweak this so instead of using style display in the div, You'll want to make it use the v-if attribute. Its a built in attribute in vue templates and you can run a conditional in there.  in the data section create a variable that is like 'leaderboardType ' and have the default value be 1. Then inside show content you can make it just change the leaderboardType to the leader board's specific value.

Then using v-if you can then inside the quotes put leaderboardType == That type value. That should have the equivalent functionality that you are looking for.

wnashif committed:
Leaderboard rework

Changed method of swapping leaderboard view.
Changed method of how data fills the leaderboard boxes.
Fixed indentation in LeaderboardsView.
Fixed a typo in HomeView.
